.github/workflows/delete-storybook-vrt-report.yml

```
@@ -0,0 +1,59 @@
+        uses: actions/checkout@v4
+        with:
+          ref: gh-pages
+
+      - name: Set Git User
+        # see: https://github.com/actions/checkout/issues/13#issuecomment-724415212
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Check for workflow reports
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:18: Double quote to prevent globbing and word splitting

(shellcheck)